src/App/index.vue

```diff
@@ -1,16 +1,40 @@
 <template lang="jade">
-  <Greeting friend="John" v-bind:stuff=stuff />
+  <div>
+    <TodoList v-bind:todos="todos"></TodoList>
+    <TodoForm @addTodo="handleTodoAdd"></TodoForm>
```

Comment:
The `@addTodo` syntax is the same as `v-on:addTodo`. This listens for the `$emit('addTodo')` event from the `TodoForm` and passes the value from the `$emit` to the `handleTodoAdd` function